OPT-6476 (#1341)
camunda/camunda-platform-docs: optimize/components/userguide/additional-features/data-source-variants.md

---
id: data-source-variants

> title: Data source variants

Reviewer: I suggest calling this `process variants comparison` because It is much more intuitive to the user to understand.

We also have a [plan](https://jira.camunda.com/browse/OPT-5584) to completely get rid of the `data source` term.

description: Using copy data source tool to compare different variants of the process
---

> When creating reports, it is possible to copy the data source and set different versions and/or tenants each copy as well as give it a custom title. This allows you to compare different variants of the process.

Reviewer: ```suggestion
When creating reports, it is possible to copy the data source and set different versions and/or tenants for each copy, as well as give it a custom title. This allows you to compare different variants of the process.
```

To use this feature, navigate to the definition edit window from inside a report. Click the copy icon while hovering over the data source. This will create a copy of the data source with all its parameters.

> Now click the pencil icon while hovering over the data source you want to edit and select version and/or tenant. This way there are two variants of the same process which can be now compared.

Reviewer: ```suggestion
Now click the pencil icon while hovering over the data source you want to edit and select a different version and/or tenant. This way there are two variants of the same process which can be now compared.
```

![Process version selection](./img/process-version-selection.png)

> Here is an exapmle of bar chart comparing average process instance duration of two versions of the same process.

Reviewer: ```suggestion
Here is an example of a bar chart report comparing the average process instance duration of two versions of the same process.
```
